xstate-chatbot/nodejs/src/machine/vitals.js

```diff
@@ -615,15 +758,18 @@ const vitalsFlow = {
               }),
               always: [
                 {
```

Comment:
Add transition to error state when intention is unknown

Reply:
Please add as first condition
cond: (context) => context.intention == dialog.INTENTION_UNKOWN,
target: 'error'

Reply:
added